src/main/java/com/faforever/api/data/domain/AbstractEntity.java

```diff
@@ -11,8 +12,8 @@
 
 @MappedSuperclass
 @Setter
+@EqualsAndHashCode(of = "id")
```

Comment:
If we additionally override the canEqual method as done in [faf-java-commons](https://github.com/FAForever/faf-java-commons/blob/6623f1f59c8bcb0d7c33423708cbf169e7c65638/faf-commons-api/src/main/java/com/faforever/commons/api/dto/AbstractEntity.java#L24) we don't need to annotate subclasses.